ocs_ci/deployment/deployment.py

```diff
@@ -1045,3 +1011,42 @@ def create_catalog_source(image=None, ignore_upgrade=False):
     )
     # Wait for catalog source is ready
     catalog_source.wait_for_state("READY")
+
+
+def setup_persistent_monitoring():
+    """
+    Change monitoring backend to OCS
+    """
+    sc = helpers.default_storage_class(interface_type=constants.CEPHBLOCKPOOL)
+
+    # Get the list of monitoring pods
+    pods_list = get_all_pods(
+        namespace=defaults.OCS_MONITORING_NAMESPACE,
+        selector=["prometheus", "alertmanager"],
+    )
+
+    # Create configmap cluster-monitoring-config and reconfigure
+    # storage class and telemeter server (if the url is specified in a
+    # config file)
+    create_configmap_cluster_monitoring_pod(
+        sc_name=sc.name,
+        telemeter_server_url=config.ENV_DATA.get("telemeter_server_url"),
+    )
+
+    # Take some time to respin the pod
+    waiting_time = 45
+    logger.info(f"Waiting {waiting_time} seconds...")
+    time.sleep(waiting_time)
```

Comment:
@clacroix12 , Instead of sleep can we wait for the resource? rest LGTM.

Reply:
This logic hasn't changed from before, I just moved it into a new function to clean up a bit. Regardless, isn't that what `validate_pods_are_respinned_and_running_state` should be doing already?

Reply:
ok so if that is the case then do we need sleep here ?

Reply:
since it was already existing , I don't want to hold this PR. LGTM.